spec/api/v2/management/users_spec.rb

```diff
@@ -100,6 +100,51 @@
       end
     end
 
+
+
+    describe 'Returns array of users as collection' do
+      let(:data) do
+        {
+          scope: :read_users
+        }
+      end
+      let(:signers) { %i[alex jeff] }
+
+      let(:do_request) do
+        post_json '/api/v2/management/users/list',
+                  multisig_jwt_management_api_v1({ data: data }, *signers)
```

Comment:
It's actually should be `multisig_jwt_management_api_v2`

Reply:
@ec done